libcodechecker/analyze/analyzers/analyzer_types.py

```diff
@@ -22,6 +21,7 @@
 from libcodechecker.analyze.analyzers import result_handler_clang_tidy
 from libcodechecker.analyze.analyzers import result_handler_plist_to_db
 from libcodechecker.analyze.analyzers import result_handler_plist_to_stdout
+from libcodechecker.cmd.cmd_line_client import setup_client
```

Comment:
In these scenarios I am more fond of having a separate pull request to pull this code out and rebase this one after that one is accepted, to avoid the excess amount of TODOs.